gensim/models/word2vec.py

```diff
@@ -240,7 +240,7 @@ def __init__(
             max_vocab_size=None, sample=1e-3, seed=1, workers=3, min_alpha=0.0001,
             sg=0, hs=0, negative=5, ns_exponent=0.75, cbow_mean=1, hashfxn=hash, epochs=5, null_word=0,
             trim_rule=None, sorted_vocab=1, batch_words=MAX_WORDS_IN_BATCH, compute_loss=False, callbacks=(),
-            comment=None, max_final_vocab=None,
+            comment=None, max_final_vocab=None, reduced_windows=True
```

Comment:
Please add a trailing comma at the end of enumerations:
https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#code-style

Reply:
Will do; sorry about that.